server/cmwell-zstore/Readme.md

```diff
@@ -0,0 +1,38 @@
+# zStore
+
+zStore is a library CM-Well uses in several modules, to keep generic key-value data in a persistent (optionally temporal), distributed manner. Underlying implementation is Cassandra.
+
+zStore features are:
+- [Self contained CQL setup](#Self-contained-CQL-setup)
+- [Simple put/get API](#Simple-put/get-API)
+- [Caching layer](#Caching-layer:-zCache)
+
+## Usages
+CM-Well uses zStore for several tasks, such as
+- Keeping large FileInfotons
+- Temporarily store large payload of a large Infoton during ingest: while the meta-data is delivered from WS to BG through Kafka, if the payload is too large it is redirected to zStore and referenced from Kafka, for the BG to populate it once consumed that Kafka message.
+- TrackingActors persistence
+- STP progress persistence
+- BG/DCC progress persistence 
+
+## Features
+
+### Self contained CQL setup
+Besides `path` and `infoton` tables CM-Well has, there's another `zstore` table in the "data2" keyspace. It's columns are `uzid`, `field` and `value` where the latter is blob. zStore object runs the `CREATE IF NOT EXISTS` CQL statement when used from any module. 
```

Comment:
I'd change "there's another zstore table" to "there's a zstore table" - otherwise it sounds like there are 2 zstore tables.
"It's columns --> "Its columns".